### PR TITLE
npm run build-languages

### PR DESCRIPTION
**Pull request by forresto**
Should be enough to follow light build directions without installing highlight.js. #68 

I didn't check in the build artifacts to make the PR cleaner.

### CODE REVIEW

**Comment by conorhastings** [comment]
@forresto can you push the AVAILABLE_LANGUAGES.md file?

File: AVAILABLE_LANGUAGES.MD [hunk 1]
```diff
@@ -0,0 +1,169 @@
+## Available `language` props 
+* 1c
```
Comment (conorhastings) [inline AVAILABLE_LANGUAGES.MD]: we'll need to special case the handling of this one since var / export name can't start with a number

Comment (forresto) [inline AVAILABLE_LANGUAGES.MD]: Oops dropped the ball. I'll special-case it.

Comment (conorhastings) [inline AVAILABLE_LANGUAGES.MD]: @forresto sounds good thanks

**Comment by forresto** [comment]
Should be good now.

**Pull request merged**